.github/workflows/comment-trigger.yml

```diff
@@ -1,9 +1,12 @@
-# Comment Trigger
+# CI Review Trigger
```

Comment:
Wondering if we should rename the file to be consistent? TIOLI

Reply:
Ah, yes, good call. Will do.